doc/thing/thing-cli.rst

```diff
@@ -85,6 +85,9 @@ absolute file path.
 
    $ knot make --dts_ovl_path {PATH}
 
+.. note:: This command will ignore if the application folder has an overlay file
```

Comment:
ignore what?

Reply:
It will ignore the overlay file placed on the application folder but this file could not exist. So, using this command if the main application folder has an overlay file it will be ignored.

Reply:
if the application folder has an overlay file this command will ignore it